requirements/base.txt

```diff
@@ -35,13 +35,13 @@ signxml==2.6.0
 #           - six
 asn1crypto==1.3.0
 attrs==19.3.0
-certifi==2019.11.28
-cffi==1.13.2
+certifi==2020.4.5.1
+cffi==1.14.0
 eight==0.4.2
 future==0.16.0
-importlib-metadata==0.23
-pycparser==2.19
-pyrsistent==0.15.7
+importlib-metadata==1.6.0
```

Comment:
It's not strictly necessary, but you may want to consider adding an environment marker (` ; python_version < "3.8"`) to this requirement (`importlib-metadata==1.6.0`). The package `jsonschema` did it.

Reply:
Good catch!